grails-app/controllers/au/org/ala/profile/api/ApiController.groovy

```diff
@@ -510,6 +513,177 @@ class ApiController extends BaseController {
+                                    @Header(name = 'Access-Control-Allow-Headers', description = "CORS header", schema = @Schema(type = "String")),
+                                    @Header(name = 'Access-Control-Allow-Methods', description = "CORS header", schema = @Schema(type = "String")),
+                                    @Header(name = 'Access-Control-Allow-Origin', description = "CORS header", schema = @Schema(type = "String"))
+                            ]
+                    ),
+                    @ApiResponse(responseCode = "400",
+                            description = "opusId and profileId are required parameters"),
+                    @ApiResponse(responseCode = "403",
+                            description = "You do not have the necessary permissions to perform this action."),
+                    @ApiResponse(responseCode = "405",
+                            description = "An unexpected error has occurred while processing your request."),
+                    @ApiResponse(responseCode = "404",
+                            description = "Opus or profile not found"),
+                    @ApiResponse(responseCode = "500",
+                            description = "An unexpected error has occurred while processing your request.")
+            ],
+            parameters = [
+                    @Parameter(name = "opusId",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "Collection id - UUID or short name"),
+                    @Parameter(name = "profileId",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "Profile id - UUID or Scientific name"),
+                    @Parameter(name = "type",
+                            in = ParameterIn.PATH,
+                            required = true,
+                            description = "type - private or open"),
+                    @Parameter(name = "Access-Token",
+                            in = ParameterIn.HEADER,
+                            required = false,
+                            description = "Access token to read private collection"),
+                    @Parameter(name = "Accept-Version",
+                            in = ParameterIn.HEADER,
+                            required = true,
+                            description = "The API version",
+                            schema = @Schema(
+                                    name = "Accept-Version",
+                                    type = "string",
+                                    defaultValue = '1.0',
+                                    allowableValues =  ["1.0"]
+                            )
+                    )
+            ],
+            security = [@SecurityRequirement(name="auth"), @SecurityRequirement(name = "oauth")]
+    )
+
+    def retrieveLocalThumbnailImage () {
+        if (!params.type) {
```

Comment:
forward request to retrieveLocalThumbnailImage of Profile controller. not need for this logic.

Reply:
changed